### PR TITLE
all changes are made.

### PR DESCRIPTION
**Pull request by keshriraj7870**

### CODE REVIEW

File: templates/front_page.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                     Projects
                 </div>
             </a>
-            <img src="{{ url_for('static', filename='projects_card_img.jpg')}}"
+           <a href="https://goo.gl/images/shx1pQ"></a>
                  class="block-img">
```
Comment (Vibhu-Agarwal) [inline templates/front_page.html]: Start tag missing

File: templates/front_page.html [hunk 2]
```diff
@@ -83,7 +83,7 @@
                     API
                 </div>
             </a>
-            <img src="{{ url_for('static', filename='api_card_img.jpg')}}"
+           <a href="https://goo.gl/images/THgnzT"></a>
                  class="block-img">
```
Comment (Vibhu-Agarwal) [inline templates/front_page.html]: Start tag missing

File: templates/front_page.html (hunk 1 above)
Comment (Vibhu-Agarwal) [inline templates/front_page.html]: Start tag missing (same in other three cards too)

Comment (keshriraj7870) [reply templates/front_page.html]: so what can i do to correct it.

File: templates/front_page.html [hunk 3]
```diff
@@ -27,7 +27,7 @@
                     Projects
                 </div>
             </a>
-            <img src="{{ url_for('static', filename='projects_card_img.jpg')}}"
+           <a href="https://goo.gl/images/shx1pQ"></a>
```
Comment (Vibhu-Agarwal) [inline templates/front_page.html]: Image links should be put in `` tag, not in the anchor `<a>` tag
Anchor tag creates a link with a text
It doesn't add image to the document

File: templates/front_page.html (hunk 1 above)
Comment (Vibhu-Agarwal) [inline templates/front_page.html]: start tag missing

**Comment by Vibhu-Agarwal** [comment]
Due to your inactivity I'm assuming that you've lost interest in the project so I'm closing your PR.
If you want to contribute, just make changes and open another PR,
I'll be happy to review and merge if it helps Marauders :)

**Pull request closed without merging**